api/src/main/java/org/door2door/dao/impl/LoginDaoImpl.java

```diff
@@ -0,0 +1,22 @@
+package org.door2door.dao.impl;
+
+import org.door2door.dao.LoginDao;
+import org.door2door.entities.TeamEntity;
+import org.hibernate.Session;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+
+public class LoginDaoImpl implements LoginDao {
+    EntityManager entityManager;
```

Comment:
I know I messed this up on the OrganizationService, but our variables for dependency injection. should be private final

private final EntityManager entityManager